sphinx-docs/usermanual/userman_admin.rst

```diff
@@ -1,18 +1,233 @@
+
+Permanently Deleting Teachers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#. Log in to KA Lite.
+#. Click on the "Manage" tab at the top of the page.
+#. Make sure that the "Facilities" tab is selected.
+#. Under the "Facilities" header, select the facility the teacher belongs to.
+#. Under the "Coaches" header, mark the box to the left of the teacher account you would like to delete.
+#. Press the "Delete Coaches" button.
+#. You will be prompted with a confirmation of your deletion. Press "OK" to proceed with the deletion.
+
+Adding a Group
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+You can create groups within a facility. Each group can represent a classroom, a study group, or any other way you would like to group students. To create a group, follow the instructions below:
+
+#. Log in to KA Lite.
+#. Click on the "Manage" tab at the top of the page.
+#. Make sure that the "Facilities" tab is selected.
+#. Select the facility that the group will belong to.
+#. Under the "Student Groups" header, click on "Add a new group".
+#. Fill out the name of the group, and provide a description.
+#. Click "create group".
+#. You should be redirected back to the page for the facility. If the group was successfully created, you will see it listed under the "Student Groups" section.
+
+Deleting a Group
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+#. Log in to KA Lite.
+#. Click on the "Manage" tab at the top of the page.
+#. Make sure that the "Facilities" tab is selected.
+#. Mark the box to the left of the group you would like to delete.
+#. Press the "Delete Groups" button under the "Student Groups" header.
+#. You will be prompted with a confirmation of your deletion. Press "OK" to proceed with the deletion.
+
+
+Moving a User to a New Group
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#. Navigate to the page for the facility the user belongs to.
+#. Under the "Students" header, select the student you would like to move by clicking in the checkbox to the left of the student name.
+#. In the dropbox, select the group you would like to move the user to.
+#. Click the "Change Student Groups" button.
+#. The page will refresh, with a message at the top indicating a successful move.
+
+Removing Users from a Group
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+If you'd like to remove a user from a group without permanently deleting the user, please follow the instructions below:
+
+#. 
+
+
+
+
+Group Summary Statistics
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For each group, you should be able to view some statistics. 
+
+#. Navigate to the Student Groups section of the facility you wish to look at. 
+#. Click on the gorpu that you wish to view.
+#. The statistics for the group should be displayed at the top of the page.
+
+Edit User Information
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
+#. Navigate to the page for the facility that the user belongs in.
+#. Find the user you would like to edit.
+#. Click the blue pencil icon next to the name of the user that you would like to edit.
```

Comment:
We could include a thumbnail of the image here, just by saving it and using the `:: image` directive.